### PR TITLE
Changelog v1.11.5

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,31 @@ This changelog goes through all the changes that have been made in each release
...
+  * Refresh codemirror on show pull comment tab (#11100) (#11122)
+  * Fix merge dialog on protected branch with missing required statuses (#11074) (#11084)
+  * Load pr Issue Poster on API too (#11033) (#11039)
+  * Fix release counter on API repository info (#10968) (#10996)
+  * Generate Diff and Patch direct from Pull head (#10936) (#10938)
+  * Fix rebase conflict detection in git 2.26 (#10929) (#10930)
+* ENHANCEMENT
+  * Fix 404 and 500 image size in small size screen (#11043) (#11049)
+* BUILD
+  * Mulitple Gitea Doctor improvements (#10943) (#10990) (#10064) (#9095) (#10991)
```
Comment (6543) [inline CHANGELOG.md]: wrong tagged?!?

Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Multiple Gitea Doctor improvements (#10943) (#10990) (#10064) (#9095) (#10991)
```

And yes, I'd move it to `ENHANCEMENT`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,31 @@ This changelog goes through all the changes that have been made in each release
...
+  * On Repo Deletion: Delete related TrackedTimes too (#11110) (#11125)
+  * Refresh codemirror on show pull comment tab (#11100) (#11122)
+  * Fix merge dialog on protected branch with missing required statuses (#11074) (#11084)
+  * Load pr Issue Poster on API too (#11033) (#11039)
+  * Fix release counter on API repository info (#10968) (#10996)
+  * Generate Diff and Patch direct from Pull head (#10936) (#10938)
+  * Fix rebase conflict detection in git 2.26 (#10929) (#10930)
+* ENHANCEMENT
+  * Fix 404 and 500 image size in small size screen (#11043) (#11049)
+* BUILD
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,31 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.5](https://github.com/go-gitea/gitea/releases/tag/v1.11.5) - 2020-05-08
+
+* BUGFIXES
+  * When delete tracked time through the API return 404 not 500 (#11319) (#11326)
+  * Prevent multiple listings of organization when creating a repository (#11303) (#11325)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Prevent duplicate records in organizations list when creating a repository (#11303) (#11325)
```
Perhaps this is more on point with the PR?

**Review by zeripath — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zeripath) [reply CHANGELOG.md]: I think the original pr was probably build but it backported quite a lot of things so it probably gains enhancement

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
Frontport income

**Pull request merged**